packages/os/os.spec

```diff
@@ -96,15 +97,12 @@ Requires: %{_cross_os}thar-be-settings
 Requires: %{_cross_os}thar-be-updates
 Requires: %{_cross_os}updog
 
-%if %{with aws_k8s_family}
-Requires: %{_cross_os}pluto
-%endif
-
 %if %{with aws_platform}
 Requires: %{_cross_os}shibaken
-Requires: %{_cross_os}cfsignal
 %endif
 
+Requires: (%{_cross_os}cfsignal if %{_cross_os}variant-platform(aws))
```

Comment:
> the user can still include these things by explicitly placing them in variant packages list?

That, or by adding this to one of their packages that's explicitly installed:
```
Provides: %{_cross_os}variant-platform(aws)
```

Having a `<ootb>-release` package to collect things like this will be a good convention.